src/test/create-user.test.ts

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+import { expect } from 'chai';
+
+describe('createUser mutation', () => {
+  it('should create a new user', async () => {
+    const expectedUser = {
+      data: {
+        name: 'Blue Pen',
+        email: 'bluepen@test.com',
+        password: 'test123',
+        birthDate: '12.02.1969',
+      },
+    };
+
+    const response = await axios.post(`http://localhost:3001/`, {
+      query: `
+      mutation CreateUser($data: UserInput) {
+          createUser(data: $data) {
+            id
+            name
+            email
+            password
+            birthDate
+          }
+        }
+      `,
+      variables: expectedUser,
+    });
+
+    expect(response.data.data.createUser.email).to.be.deep.eq(expectedUser.data.email);
+    expect(response.data.data.createUser.name).to.be.deep.eq(expectedUser.data.name);
+    expect(response.data.data.createUser.birthDate).to.be.deep.eq(expectedUser.data.birthDate);
```

Comment:
`deep` normalmente é utilizado para comparar objetos/array, como vc ta comparando valores direto n precisa usar `deep` né?
```suggestion
    expect(response.data.data.createUser.email).to.be.eq(expectedUser.data.email);
    expect(response.data.data.createUser.name).to.be.eq(expectedUser.data.name);
    expect(response.data.data.createUser.birthDate).to.be.eq(expectedUser.data.birthDate);
```

Reply:
No caso eu recomendaria usar o deep mesmo, mas para comparar todo o objeto de resposta. Isso vai garantir que ele vai checar cada chave e valor do objeto de resposta, e vai falahr se tiver chave a menos ou a mais.